### PR TITLE
New FortiClient-VPN cask, different from FortiClient

### PR DESCRIPTION
**Pull request by david-ns**
This cask adds support for FortiClient VPN, which is a different software than FortiClient (already available as a cask)

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).
- [x] `brew audit --cask {{cask_file}}` is error-free.
- [x] `brew style --fix {{cask_file}}` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew install --cask {{cask_file}}` worked successfully.
- [x] `brew uninstall --cask {{cask_file}}` worked successfully.

Additional information:
This is the original cask named forticlient, which recently changed the software from forticlient-vpn to forticlient (another software from the same company) due to cask update #99090 (originating confusion among users of the cask). Since it makes sense from the naming perspective, I created a new cask for forticlient-vpn and I suggest keeping FortiClient on the forticlient cask.

### CODE REVIEW

File: Casks/forticlient-vpn.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+cask "forticlient-vpn" do
+  version "6.4"
+  sha256 "a673cc372f4caf90476e6cda4e51c8450ac58e09a8c5218bba537e86792e5d23"
+
+  url "https://filestore.fortinet.com/forticlient/downloads/FortiClientVPNOnlineInstaller_#{version}.dmg",
```
Comment (vitorgalvao) [inline Casks/forticlient-vpn.rb:5]: “Online installer” typically means it downloads more stuff later. Isn’t there an offline installer which contains everything?

File: Casks/forticlient-vpn.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      verified: "filestore.fortinet.com/forticlient/"
+  name "FortiClient VPN"
+  desc "Free VPN client for FortiClient"
+  homepage "https://forticlient.com/"
+
+  installer manual: "FortiClientUpdate.app"
+
+  uninstall script: {
+    executable: "/Applications/FortiClientUninstaller.app/Contents/Resources/uninstall_helper",
```
Comment (vitorgalvao) [inline Casks/forticlient-vpn.rb:14]: Won’t this invoke a GUI?

**Review by vitorgalvao — DISMISSED** [review]

**Review by vitorgalvao — CHANGES_REQUESTED** [review]
See comments

File: Casks/forticlient-vpn.rb (hunk 1 above)
Comment (David-NS) [reply Casks/forticlient-vpn.rb:5]: I'm afraid there is not an offline installer for the FortiClient VPN package (free to use). I've made a quick search and they offer one for the paid client, but nothing for the free one.

File: Casks/forticlient-vpn.rb (hunk 2 above)
Comment (David-NS) [reply Casks/forticlient-vpn.rb]: I've double-checked it and it invoques a confirmation window. Also, I've seen that at some point the uninstaller location have changed from Resources/uninstall_helper to MacOS/Uninstall.

**Review by david-ns — COMMENTED** [review]
I've searched online and I'm afraid they only provide an offline installer for the paid version of FortiClient.
The uninstaller indeed launches a confirmation dialog, but the path was wrong. I've made a commit fixing it.

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**